base/db/tests/Rcheck_reference.log

```diff
@@ -34,98 +90,49 @@ Requires orphaned package: ‘udunits2’
 * checking whether the namespace can be loaded with stated dependencies ... OK
 * checking whether the namespace can be unloaded cleanly ... OK
 * checking loading without being on the library search path ... OK
-* checking dependencies in R code ... OK
+* checking use of S3 registration ... OK
+* checking dependencies in R code ... NOTE
+Namespaces in Imports field not imported from:
+  ‘glue’ ‘knitr’
+  All declared Imports should be used.
 * checking S3 generic/method consistency ... OK
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
 * checking R code for possible problems ... NOTE
-
-
 db.exists: no visible binding for '<<-' assignment to ‘user.permission’
 db.exists: no visible binding for global variable ‘user.permission’
-
-load_data_single_run: no visible global function definition for
-  ‘read.output’
 load_data_single_run: no visible binding for global variable ‘var_name’
 load_data_single_run: no visible binding for global variable ‘vals’
-match_dbcols: no visible global function definition for ‘head’
 match_dbcols: no visible binding for global variable ‘.’
 match_dbcols: no visible binding for global variable ‘as’
-
 query.priors: no visible binding for global variable ‘settings’
-
-
-
 Undefined global functions or variables:
-  . as author author_family author_given citation_id collect
-  container_id container_type created_at cultivar_id ensemble_id folder
-  genus greenhouse head id issued machine_id n name name.cv name.mt
-  pft_id pft_type posix read.output run_id scientificname score
-  settings site_id specie_id species stat storage_type sync_host_id
-  trait trt_id updated_at user.permission vals var_name variable_id
-  workflow_id
+  . as settings user.permission vals var_name
```

Comment:
I'm assuming that you'll be working on removing the last few globals?

Reply:
Yes, I was clearing the `PEcAn.MA` package first, Now I will clear the `PEcAn.DB` package